tests/visual/percy-storybook/Border.defaults.percy.tsx

```diff
@@ -3,6 +3,7 @@ import React from 'react';
 import { storiesOf } from '@storybook/react';
 import random from 'core/math/random';
 import DataTable from 'dash-table/dash/DataTable';
+import './common';
```

Comment:
Could technically add this to only one test and it would be applied to all but it seemed better to duplicate the import just so it's clear that this is "expected".